integration-tests/node-typescript/nodets_test.go

```diff
@@ -0,0 +1,182 @@
+	require.NotEmpty(t, npmArgs)
+
+	cmd := exec.Command("npm", npmArgs...)
+	cmd.Dir = projectDir
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+
+	t.Logf("Command: %s", cmd.String())
+	err := cmd.Run()
+	require.NoError(t, err)
+
+	// Create jest.config.ts file
+	jestConfig := getJestConfig(t, instructions)
+	jestConfigPath := filepath.Join(projectDir, "jest.config.ts")
+	err = os.WriteFile(jestConfigPath, []byte(jestConfig), 0o644)
+	require.NoError(t, err)
+
+	// Execute the create command
+	fuzzTestPath := filepath.Join(projectDir, "FuzzTestCase.fuzz.ts")
+	cifuzzRunner.CommandWithFilterForInstructions(t, "create", &shared.CommandOptions{
+		Env:  append(os.Environ(), "CIFUZZ_PRERELEASE=1"),
+		Args: []string{"ts", "--output", fuzzTestPath},
+	},
+	)
+
+	// Check that the fuzz test was created in the correct directory
+	require.FileExists(t, fuzzTestPath)
+
+	// Check that the findings command doesn't list any findings yet
+	findings := shared.GetFindings(t, cifuzz, projectDir)
+	require.Empty(t, findings)
+
+	// Run the (empty) fuzz test
+	cifuzzRunner.Run(t, &shared.RunOptions{
+		ExpectedOutputs:              []*regexp.Regexp{regexp.MustCompile(`^paths: \d+`)},
+		TerminateAfterExpectedOutput: true,
+	})
+
+	// Make the fuzz test call a function
+	modifyFuzzTestToCallFunction(t, fuzzTestPath)
+	// Run the fuzz test
+	expectedOutputExp := regexp.MustCompile("Crash!")
+	// setting -max_len to 8192 in ".jazzerjsrc" file to test if the file is read and picked up
+	unexpectedOutputExp := regexp.MustCompile("INFO: -max_len is not provided; libFuzzer will not generate inputs larger than 4096 bytes")
+	cifuzzRunner.Run(t, &shared.RunOptions{
+		ExpectedOutputs:  []*regexp.Regexp{expectedOutputExp},
+		UnexpectedOutput: unexpectedOutputExp,
+	})
+
+	// Check that the findings command lists the finding
+	findings = shared.GetFindings(t, cifuzz, projectDir)
+	require.Len(t, findings, 1)
+	require.Contains(t, findings[0].Details, "Crash!")
+
+	expectedStackTrace := []*stacktrace.StackFrame{
+		{
+			SourceFile:  "ExploreMe.ts",
+			Line:        6,
+			Column:      12,
+			FrameNumber: 0,
+			Function:    "exploreMe",
+		},
+	}
+	require.Equal(t, expectedStackTrace, findings[0].StackTrace)
+}
+
+func getNpmArgs(t *testing.T, instructions []string) []string {
+	t.Helper()
+
+	for _, instruction := range instructions {
+		if strings.HasPrefix(instruction, "npm") {
+			args := strings.TrimSpace(strings.TrimPrefix(instruction, "npm"))
+			return strings.Split(args, " ")
+		}
+	}
+
+	return nil
+}
+
+func getJestConfig(t *testing.T, instructions []string) string {
+	t.Helper()
+
+	for i, instruction := range instructions {
+		if strings.HasPrefix(instruction, "import type") {
+			return strings.Join(instructions[i:len(instructions)-1], "\n")
+		}
+	}
+
+	return ""
+}
+
+func modifyFuzzTestToCallFunction(t *testing.T, fuzzTestPath string) {
+	// Modify the fuzz test stub created by `cifuzz create` to actually
+	// call a function.
```

Comment:
Should these comments be above the function name to be picked up by IDEs?
```suggestion
// Modify the fuzz test stub created by `cifuzz create` to actually call a function.
func modifyFuzzTestToCallFunction(t *testing.T, fuzzTestPath string) {
```